libs/plugin-types/index.d.ts

```diff
@@ -194,6 +194,70 @@ export interface PluginData {
   getSharedPluginDataKeys(namespace: string): string[];
 }
 
+/**
+ * Comments allow the team to have one priceless conversation getting and
+ * providing feedback right over the designs and prototypes.
+ */
+export interface Comment {
+  /**
+   * The `user` that has created the comment.
+   */
+  readonly user: User;
+
+  /**
+   * The `date` the comment has been created.
+   */
+  readonly date: Date;
+
+  /**
+   * The `text` for the commentary. The owner can modify the comment.
+   */
+  text: string;
+
+  /**
+   * Remove the current comment from its comment thread.
+   */
+  delete(): void;
+}
+
+/**
+ * Represents a list of comments one after the other. Usually this threads
+ * are conversations the users have in Penpot.
+ */
+export interface CommentThread {
+  /**
+   * If the thread is attached to a `board` this will have that board
+   * reference.
+   */
+  board?: Board;
+
+  /**
+   * The `position` in absolute coordinates in the canvas.
+   */
+  position: Point;
+
+  /**
+   * List of `comments` ordered by creation date.
+   */
+  comments: Comment[];
+
+  /**
+   * Whether the thread has been marked as `resolved` or not.
+   */
+  resolved: boolean;
+
+  /**
+   * Creates a new comment after the last one in the thread. The current user will
+   * be used as the creation user.
+   */
+  reply(text: string): Comment;
+
+  /**
+   * Removes the current comment thread.
```

Comment:
Same here, anyone can remove the comment or just the current user who has created it?

Reply:
Changed.